capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module

```diff
@@ -165,8 +169,19 @@ function c4m_restful_quick_post_c4m_location_field_entity_property_info(&$info,
 function c4m_restful_quick_post_preprocess_html(&$vars) {
   $item = menu_get_item();
 
-  if (!menu_get_object() && isset($item['map']) && $item['map'][0] != 'node') {
-    // Not a node page and not a node/add(edit) page.
+  $allowed_paths = array(
+    'node/add',
+    'node/%/edit',
+    'node',
+  );
+
+  if (!in_array($item['path'], $allowed_paths)) {
+    // Not a group dashboard page and not a node/add(edit) page.
+    return;
+  }
+  if ($item['path'] == 'node' && !og_context()) {
```

Comment:
Is there a good reason not to use the [drupal_is_front_page()](https://api.drupal.org/api/drupal/includes%21path.inc/function/drupal_is_front_page/7) function?

```
if (drupal_is_front_page() && !og_context()) {
  ...
}
```